payments/utils.py

```diff
@@ -0,0 +1,84 @@
+from datetime import timedelta
+from decimal import Decimal
+import logging
+
+from django.db.models import Q
+from django.conf import settings
+from django.utils import timezone
+
+from fiobank import FioBank
+
+from konfera.models import Order
+from konfera.models.order import AWAITING, PAID, PARTLY_PAID
+from payments.models import ProcessedTransaction
+
+
+DATE_FORMAT = '%Y-%m-%d'
+
+
+logger = logging.getLogger(__name__)
+
+
+def _get_last_payments():
+    """ Get list of payments for last three days from FioBank """
+    client = FioBank(token=settings.FIO_BANK_TOKEN)
```

Comment:
this is third party connector (either here or in period method), I am just curious what happen if lets say Fio is down, or our server is running but offline...

I just had a thought what if we modify it to be executable via button (lets say we have button in admin area, process FIO payments) and if FIO is down for maintance for example, we want to show nice message instead of exception... How about we wrap it in try catch block and log all exceptions, this way we will have data in case problem in FioBank occurs, for the moment we can have exception at least we will test the third party app a little before  real usage...